_data/simple-icons.json

```
@@ -9345,6 +9345,11 @@
             "hex": "101010",
             "source": "https://www.mazda.com/en/about/profile/library/"
         },
+        {
+            "title": "Maze",
+            "hex": "000000",
+            "source": "https://maze.co"
```

Press kit linked in the footer.
```suggestion
            "source": "https://mazedesign.notion.site/Press-Kit-421d405e76154662a547ce9ef2a3f018"
```